File: nodejs/ch02/2525.js
const fs = require('fs');
const input = fs.readFileSync("../dev/stdin").toString().trim().split(/\s+/).map(Number);

let a = input[0]
let b = input[1] + input[2]



if (b >= 60) {
    a = a + Math.floor(b / 60)
    b = b % 60
    if (a >= 24) {
        a = a - 24
    }
    console.log(a, b);
} else {
    if (a >= 24) {
        a = a - 24
    }
    console.log(a, b)
}

File: python/ch02/2525.py
h, m = input().split()

time = int(input())
h, m = int(h), int(m)

time_h = time // 60
time_m = time % 60

# 60 안넘을 경우
if time < 60:

    # 60분이 넘을 경우
    if m + time_m > 60:
        h += 1
        m = 60 - (m + time_m)
    # 60분 안넘을 경우
    else:
        m += time_m
# 60분 넘을 경우
else:
    h += time_h
    m += time_m

if h >= 24:
    h -= 24

if m >= 60:
    h += 1
    m -= 60

print(h,m)